lib/internal/bootstrap_node.js

```diff
@@ -504,7 +513,7 @@
   };
 
   NativeModule.wrapper = [
-    '(function (exports, require, module, __filename, __dirname) { ',
+    '(function (exports, require, module, __filename, process) { ',
```

Comment:
I assumed it knew its current module using some other mechanism, but that makes sense. Also about __dirname.